src/app/public/modules/design-guidelines/design-guideline-thumbnail.component.scss

```diff
@@ -3,18 +3,6 @@
 
 .sky-docs-design-guideline-thumbnail {
   margin-bottom: $sky-margin * 5;
-
-  ::ng-deep {
```

Comment:
These changes are no longer needed because they were applied to the `skyux-lib-media` library.